app.js

```diff
@@ -36,7 +51,8 @@ app.get("/horoscopes", function (req, res){
             console.log(err);
         } else {
             allHoroscopes.reverse();
-            res.render("horoscopes/index", {allHoroscopes:allHoroscopes});
+            firstTenHoroscopes = allHoroscopes.slice(0,10)
+            res.render("horoscopes/index", {firstTenHoroscopes:firstTenHoroscopes});
```

Comment:
hmmmmmm I think I'mma go with the longhand, if only for clarity's sake :) I tend to avoid shorthand where possible so my code looks less magical :) :) :)